src/provisioner/const.py

```diff
@@ -25,3 +25,7 @@
 # 'UPGRADE_MODE_VAL' constant for default upgrade mode.
 UPGRADE_MODE_VAL = ""
 REQUIRED_EXTERNAL_SW = ['kafka', 'consul']
+DELTA_PATH = "yaml:///etc/cortx/delta.conf"
```

Comment:
We should avoid references to path, or file, etc. It should only be called URL. No file system operations can be performed on the URL

Reply:
done